src/codegen/mod.rs

```diff
@@ -1918,8 +1918,10 @@ impl CodeGenerator for CompInfo {
                 self.kind(),
             );
 
+            let fmtparent = if ctx.options().use_core { quote! { core } } else { quote! { std } };
```

Comment:
`ctx.trait_prefix()` instead, and let's call this `prefix` for consistency with other `codegen` code.

Reply:
Thanks, done; I was looking at options for such a method.